src/components/events/Events.js

```diff
@@ -292,7 +292,7 @@ const Events = () => {
               </Text>
 
               <Text className="body to-text" marginTop="8px">
-                Date: {new Date(new Date(event.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleString()}
+                Date: {new Date(new Date(event.date).getTime() - (4 * 60 * 60 * 1000)).toLocaleString()}
```

Comment:
just realized this: for events, the timestamp itself (ex: 8:00PM) isn't important. Can you update the `toLocaleString()` method to `toLocaleDateString()`? You need to do the same for My Events